evap/rewards/tools.py

```diff
@@ -79,19 +79,19 @@ def grant_reward_points_if_eligible(user, semester):
     missing_points = target_points - granted_points
 
     if missing_points > 0:
-        RewardPointGranting.objects.create(user_profile=user, semester=semester, value=missing_points)
-        return missing_points, progress >= 1.0
+        granting = RewardPointGranting.objects.create(user_profile=user, semester=semester, value=missing_points)
+        return granting, progress >= 1.0
     return 0, False
```

Comment:
now i'd prefer to return None instead of all the zeroes to keep the types consistent.